docs/assets/css/src/docs.css

```diff
@@ -28,6 +28,11 @@ body {
   font-weight: normal;
 }
 
+/* In-line code within headings retain the heading background */
```

Comment:
It's `inline` and not `in-line` as far as I can tell.

Also, break one selector per line.